pwiz_tools/Shared/CommonUtil/Chemistry/MzTolerance.cs

```diff
@@ -85,6 +85,17 @@ public bool IsWithinTolerance(double a, double b)
             return (a >= b - this) && (a <= b + this);
         }
 
+        public int CompareWithTolerance(double a, double b)
```

Comment:
I would recommend removing this method because it's entirely obvious just from the method's name what exactly it's going to do.
If you were going to keep this method, I think its implementation should make use of `IsWithinTolerance`:
```
        public int CompareWithTolerance(double a, double b)
        {
            if (IsWithinTolerance(a, b))
            {
                return 0;
            }

            return a < b ? -1 : 1;
        }
```